hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -71,6 +71,9 @@
     @Probe(name = "connectionListenerCount")
     final Set<ConnectionListener> connectionListeners = new CopyOnWriteArraySet<ConnectionListener>();
 
+    @Probe(name = "acceptedSocketCount", level = MANDATORY)
+    final Set<Channel> acceptedChannels = newSetFromMap(new ConcurrentHashMap<Channel, Boolean>());
```

Comment:
I would rather see it private and a method provided to remove the channel. We should avoid disclosing implementation details when possible.

Reply:
Done